### ISSUE TITLE
segmentation fault when run with no arguments

### ISSUE THREAD

**Issue by RioNight** [shilangyu/scoop-search#49]
I get a segmentation fault when running scoop-search with no arguments.

I'm using the bash shell which comes with Git for Windows, if that matters.

![image](https://github.com/shilangyu/scoop-search/assets/58403689/c92d2412-2ee9-497d-8ccd-498fb2bebb0b)

**Comment by RioNight** [comment]
It might also be worth mentioning that running scoop-search in powershell or cmd with no arguments makes scoop-search hang for a few seconds and then results in no output. I don't know if this is intended. I would expect it to exit right away.

**Comment by shilangyu** [comment]
That is correct! I actually already fixed it a week ago in https://github.com/shilangyu/scoop-search/pull/48, but forgot to merge and release it. Thank you for bumping this. I will get it sorted.

**Comment by RioNight** [comment]
Thanks.

**Comment by shilangyu** [comment]
Fixed in v1.4.1
